core/notifications/notifications.go

```diff
@@ -0,0 +1,126 @@
+package notifications
+
+import (
+	"fmt"
+
+	"github.com/anyproto/any-sync/app"
+
+	"github.com/anyproto/anytype-heart/core/event"
+	"github.com/anyproto/anytype-heart/pb"
+	"github.com/anyproto/anytype-heart/pkg/lib/pb/model"
+)
+
+const CName = "notificationService"
+
+type Notifications interface {
+	app.Component
+	CreateAndSendLocal(notification *model.Notification) error
```

Comment:
consider increasing data integrity and encapsulation by assigning id and timestamp to notification inside service